stories/Molecules/Navbar/Breadcrumb/breadcrumb-link.scss

```diff
@@ -0,0 +1,81 @@
+@import '../../../assets/scss/breakpoints';
+@import '../../../assets/scss/variables';
+@import '../../../assets/scss/mixins';
+
+.breadcrumb__wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: baseline;
+  font-size: 12px;
+  padding: 0;
+
+  li {
+    display: inline-block;
+    margin: 10px;
+    position: relative;
+
+    &::after {
+      content: '/';
+      font-weight: bold;
+      color: $color-dark-red;
+      position: absolute;
+      right: -11px;
+      top: 0;
+    }
+    span {
+      &.last-item {
+        color: $color-black;
+        display: block;
+        font-family: $Font-Family-ProximaSemiBold;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        text-transform: uppercase;
+        white-space: nowrap;
+        width: 60px;
+  
+        @include devicebreak(medium) {
+          overflow: inherit;
+        }
+      }
+    }
+
+    &:last-child::after {
+      content: none;
+    }
+  }
+
+  &--white {
+    background-color: $color-gray-400;
+
+    li {
+      &::after {
+        color: $color-white;
+      }
+
+      span {
+        &.last-item {
+          color: $color-white;
```

Comment:
@pallavikadam-srijan we can just use the class not use the tag name with it

Reply:
Done

Reply:
@pallavikadam-srijan I can still see this issue

Reply:
done